anomalib/models/dfm/lightning_model.py

```diff
@@ -100,9 +104,9 @@ def validation_step(self, batch, _):  # pylint: disable=arguments-differ
           batch (List[Dict[str, Any]]): Input batch
 
         Returns:
-          Dictionary containing FRE anomaly scores and ground-truth.
+          Dictionary containing FRE anomaly scores and anomaly maps.
         """
-        batch["pred_scores"] = self.model(batch["image"])
+        batch["pred_scores"], batch["anomaly_maps"] = self.model(batch["image"])
```

Comment:
This needs to be assigned based on the score type. If I select `nll` score type then training fails.